build/build-extensions.ps1

```diff
@@ -155,6 +158,16 @@ function CreateSiteExtensions() {
     Remove-Item $siteExtensionPath -Recurse -Force > $null
 }
 
+function WriteHashesFile([string] $directoryPath) {
+  New-Item -Path "$directoryPath\..\temp_hashes" -ItemType Directory
```

Comment:
This is just optional -- if you'd like to omit output from the `New-Item` call, you can pipe that to `Out-Null`. 

The command would look like this:
`New-Item -Path "$directoryPath\..\temp_hashes" -ItemType Directory | Out-Null`

Also, is this code only going to run on Windows? Is this is not the case, please consider using `'/'` instead of `'\'`